packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -20,322 +20,308 @@ import {
+        [body, shouldRerenderDependency, editing, appBridge, id]
+    );
 
-        return (
-            <div
-                ref={ref}
-                className="tw-bg-base"
-                style={{ ...transformStyle, ...(!isDragging ? { zIndex: undefined } : {}) }}
-            >
-                <BlockItemWrapper
-                    isDragging={isDragging}
-                    shouldHideWrapper={replaceWithPlaceholder || !editing}
-                    shouldHideComponent={replaceWithPlaceholder}
-                    shouldBeShown={isDragging}
-                    toolbarItems={[
-                        { type: 'dragHandle', icon: <IconArrowMove16 />, draggableProps, setActivatorNodeRef },
-                        { type: 'button', icon: <IconTrashBin16 />, tooltip: 'Delete Item', onClick: onRemoveSelf },
-                        {
-                            type: 'menu',
-                            items: [
-                                [
-                                    ...(!!linkedImage
-                                        ? [
-                                              {
-                                                  title: 'Replace with upload',
-                                                  icon: <IconArrowCircleUp20 />,
-                                                  onClick: onUploadClick,
-                                              },
-                                              {
-                                                  title: 'Replace with asset',
-                                                  icon: <IconImageStack20 />,
-                                                  onClick: onOpenAssetChooser,
-                                              },
-                                          ]
-                                        : []),
-                                    {
-                                        title: type === DoDontType.Do ? 'Change to "don\'t"' : 'Change to "do"',
-                                        icon: <IconArrowSwap20 />,
-                                        onClick: () =>
-                                            onChangeItem(
-                                                id,
-                                                type === DoDontType.Do ? DoDontType.Dont : DoDontType.Do,
-                                                'type'
-                                            ),
-                                    },
-                                ],
-                                [
-                                    {
-                                        title: 'Delete',
-                                        icon: <IconTrashBin20 />,
-                                        onClick: onRemoveSelf,
-                                    },
-                                ],
+    return (
+        <div className="tw-bg-base tw-relative" style={{ ...(!isDragging ? { zIndex: undefined } : {}) }}>
```

Comment:
Is this style actually doing anything anymore?

Reply:
Nope, removed